README.md

```diff
@@ -610,7 +597,7 @@ This command will allow the holder of the transferable record to endorse the tra
 **This command will fail if there is no approved owner or holder record on the transferable record.**
 
 ```bash
-open-attestation title-escrow endorse-transfer-owner --token-registry <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> [options]
+open-attestation title-escrow endorse-transfer-owner --token-registry <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --newOwner <NEW_OWNER> [options]
```

Comment:
this too, can change to newBeneficiary? it will be much easier to understand right?